spec/docx/smoke/api_stroke_spec.rb

```diff
@@ -6,4 +6,22 @@
     docx = builder.build_and_parse('js/docx/smoke/api_stroke/get_class_type.js')
     expect(docx.elements[1].nonempty_runs.first.text).to eq('Class Type = stroke')
   end
+
+  it 'ApiStroke | ToJSON method' do
+    docx = builder.build_and_parse('js/docx/smoke/api_stroke/to_json.js')
+    expect(docx.elements.first.nonempty_runs.first.alternate_content.office2010_content.graphic.data.text_body.elements.first.character_style_array.size).to eq(2)
+    expect(docx.elements.first.nonempty_runs.first.alternate_content.office2010_content.graphic.type).to eq(:shape)
+    expect(docx.elements[1].nonempty_runs.first.text).to include('"type":"stroke"')
+    stroke = JSON.parse(docx.elements[1].nonempty_runs.first.text)['graphic']['spPr']['ln']
+    expect(stroke['w']).to be_a(Integer)
+    expect(stroke['algn']).to be_nil
+    expect(stroke['fill']['fill']['color']['color']['rgba']['red']).to eq(51)
```

Comment:
Maybe use expect color obj like https://github.com/ONLYOFFICE/doc-builder-testing/blob/32b2f8a897a37be7dffe687499097d38b38d8424/spec/docx/smoke/api_style_spec.rb#L37

Reply:
Fix in the new commit.